.devcontainer/post-install.sh

```diff
@@ -39,7 +39,7 @@ dotnet tool install -g webvalidate --version 1.0.7.3
 
 # copy launch.json from docs/vscode-template to .vscode folder
 mkdir .vscode
-cp cp docs/vscode-template/* .vscode
```

Comment:
need to check the other repos to make sure this mistake isn't in there

Reply:
Fixed this command to use mkdir -p .vscode && cp docs/vscode-template/* .vscode
to matct to the other repos.